### DISCUSSION TITLE
features are missing between vtkLocalView and vtkRemoteView

### DISCUSSION THREAD

**Discussion by rrudy9** [Kitware/trame#158] [Trame/VTK] [answered]
Hello Sébastien, @jourdain 

I want to use this technique (https://kitware.github.io/vtk-examples/site/Python/CompositeData/CompositePolyDataMapper/ ) so I can be able to control my blocks in a really easy way.

It works with vtkRemoteView (bcz I guess It's the vtkWindow that is being renderer on the server and being stream). However, It doesn't work with vtkLocalView and many other features that I came across while I was developing my application. 

Do I need to activate something in the vtkLocalView? 

 (I already implement another solution but I know that there is a better solution! ;) (There is always a better solution hihihi :DDDD ;) ) 

P.S: I found a bug in this release (if there was any, or if it is a bug I don't know! ) but I have this error message with VtkLocalView --> 

"🔴 Can not forward events without a current renderer on the interactor." 

and the view is not showing! I don't know if It is a bug man! but what I did to solve it for now is to create a "refresh" function inside this class and call it to force the update 

Thank you for your help and for this great Framework! Amazing Job!!!  Thanks you again and your team ! ;) 

Roudy

**Comment by jourdain** [comment, marked as answer]
Hi Roudy,

multi-blocks are not properly supported with local view. But I'm guessing you are hitting a corner case of `trame-vtk==2.0.9`. Try using `trame-vtk==2.0.8` instead.

**Comment by rrudy9** [comment]
Hello Sébastien, 

Thank you for your answer! 

Yeah that's what I thought , too bad! :/ 

Yup, the little reload button is fire tho haha ! 

Thank you for your work! and keep it up! ;)
